### PR TITLE
Leveraging the test-driven-development lab instead of the project-tem…

### PR DESCRIPTION
**Pull request by worldwidewebbz**
…plate for the development env

### CODE REVIEW

File: service/error_handlers.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def request_validation_error(error):
 
 @app.errorhandler(status.HTTP_400_BAD_REQUEST)
 def bad_request(error):
-    """Handles bad requests with 400_BAD_REQUEST"""
+    """Handles bad reuests with 400_BAD_REQUEST"""
```
Comment (lpk229) [inline service/error_handlers.py]: requests is spelled wrong

File: service/error_handlers.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def not_found(error):
 
 @app.errorhandler(status.HTTP_405_METHOD_NOT_ALLOWED)
 def method_not_supported(error):
-    """Handles unsupported HTTP methods with 405_METHOD_NOT_SUPPORTED"""
+    """Handles unsuppoted HTTP methods with 405_METHOD_NOT_SUPPORTED"""
```
Comment (lpk229) [inline service/error_handlers.py]: unsupported is spelled wrong

File: service/error_handlers.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def method_not_supported(error):
 
 @app.errorhandler(status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
 def mediatype_not_supported(error):
-    """Handles unsupported media requests with 415_UNSUPPORTED_MEDIA_TYPE"""
+    """Handles unsuppoted media requests with 415_UNSUPPORTED_MEDIA_TYPE"""
```
Comment (lpk229) [inline service/error_handlers.py]: unsupported spelled wrong here too

**Review by lpk229 — CHANGES_REQUESTED** [review]
Mostly looks good, there are a few spelling mistakes in the code - see comments.

**Review by lpk229 — APPROVED** [review]
Everything looks good!

**Review by sb8414 — APPROVED** [review]
Looks good to me!

**Pull request merged**